src/Microsoft.DotNet.Build.Tasks/PackageFiles/ApiCompat.runtimeconfig.json

```diff
@@ -0,0 +1,8 @@
+{
+  "runtimeOptions": {
+    "framework": {
+      "name": "Microsoft.NETCore.App",
+      "version": "1.0.0-rc3-002733"
```

Comment:
I'm still debating on having this runtimeconfig files includded in the package or if they should be created using init-tools instead. The reason is that this version is tied to the framework that comes with the cli, which each repo can have a different one, but also buildtools won't work without them so delegating the creation of the files to a different repo is also not ideal. @weshaggard what do you think?